src/SDKs/ServiceBus/data-plane/src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -15,8 +15,11 @@
 
   <ItemGroup>
     <Compile Remove="obj\**" />
+    <Compile Remove="bin\**" />
```

Comment:
I wouldn't expect any of these obj or bin folder excludes to be necessary. We redirect the outputs to <repo>\artifacts\bin|obj.